src/components/docs-page/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { ReactElement } from 'react'
+import { MDXRemote } from 'next-mdx-remote'
+import defaultMdxComponents from '@hashicorp/platform-docs-mdx'
+import Placement from 'components/author-primitives/shared/placement-table'
+import NestedNode from 'components/author-primitives/waypoint/nested-node'
+
+const additionalComponents = defaultMdxComponents('waypoint', {
```

Comment:
💭 Stating the obvious here, but this will eventually need to be configurable by product.

Reply:
Agreed, I'll add a TODO so that anyone who stops by will know we're thinking of it 😊